client/lib/analytics/index.js

```diff
@@ -19,6 +19,8 @@ var config = require( 'config' ),
 	_superProps,
 	_user;
 
+import { retarget } from 'lib/analytics/ad-tracking';
```

Comment:
yeah I got it to this point, but also commented out basically all the changes from this PR in that file and it still failed the tests. not sure why.

I tried commenting out everything and it still failed.

I think there's some async thing going on. somewhere `config.isEnabled` got turned into undefined. Also, I hooked up a debugger to the test session and it never halted on that exception for some reason. I couldn't get it to.